### ISSUE TITLE
Update?

### ISSUE THREAD

**Issue by dindrala** [gt-rios/firefox-syntax#3]
Your color syntax theme is different than the current Quantum  JavaScript scratchpad. Any plans on updating to match the current color theme?  That would be amazing. Love the scratchpad's syntax theme.

**Comment by gt-rios** [comment]
Hi dindrala, the default themes are actually based on Quantum so I'm not sure what you mean:

- Do you mean the actual colors you're seeing aren't the correct ones?  
If so, you can go to __Settings > Themes > Firefox Syntax > Theme Settings__, and change the DevTools theme.

- Or do you mean colors are the right ones but the highlighting isn't correct?  
If so, go to __Settings > Core__ and check that the Tree Sitter Parser isn't enabled.

I hope this helps.

**Comment by dindrala** [comment]
The color syntax doesn’t match the color syntax theme in the current Quantum browser JavaScript scratch pad. I’ve tried all the Firefox themes in settings—-themes—-Firefox syntax.

**Comment by dindrala** [comment]
Ive included an image of the desired syntax color them I am referring to here: 


<img width="805" alt="firefox" src="https://user-images.githubusercontent.com/16546748/42856655-01d83afe-89fb-11e8-8685-abd083baf33d.png">

**Comment by dindrala** [comment]
Ive included an image of the color theme I am referring to below:

<img width="806" alt="firefox" src="https://user-images.githubusercontent.com/16546748/42856710-48b0d6de-89fb-11e8-95a9-df4bdf59a549.png">

**Comment by gt-rios** [comment]
Yes, that color scheme corresponds to the `Default Dark` theme of this package. Here's the theme working inside Atom with the same code as your screenshot:
![](https://user-images.githubusercontent.com/17343833/42860681-4b1c370c-8a1d-11e8-8f53-9145bb290665.jpg)
Can you confirm that this is what you see when you go to __Settings > Themes > Firefox Syntax > Theme Settings__ and select `Default Dark`? The color scheme is the same but there are some parts that aren't highlighted because the parser doesn't assign any classes to them, although I believe I can style `document` to be purple and some occurrences of `str` and `x` to be blue in the next release, so thanks for bringing that to my attention.

**Comment by dindrala** [comment]
@gt-rios Appreciate you getting back to me. **Yes I can confirm** that the color syntax you posted is what i see in Settings > Themes > Firefox Syntax > Theme Settings > Default Dark. Yup, it looks like variable and arguments are baby blue, objects are all purple, methods lime green and key words pink.   That would be great! I'm really liking this new color theme. Can't wait for the next release. And you're welcome.

**Comment by dindrala** [comment]
<img width="805" alt="ff" src="https://user-images.githubusercontent.com/16546748/42916205-db344118-8ab8-11e8-98ba-eda38879a9f1.png">

**Comment by dindrala** [comment]
noticed str (variables) are purple in my image. that's odd and incorrect. ive posted another image below. i took this screenshot just now of the JS scratchpad.

<img width="805" alt="ff" src="https://user-images.githubusercontent.com/16546748/42916253-2973f9cc-8ab9-11e8-8a57-7c6df3c3a107.png">

**Comment by dindrala** [comment]
@gt-rios Hi. Any eta on the update? Also, would availability for Brackets be possible?

**Comment by gt-rios** [comment]
@dindrala Yes, **v4.2.0** was released a few days ago with a complete rewrite of the JavaScript syntax, and more improvements on the way; the goal is to make it look like the Scratchpad as much *as possible* (considering the parser limitations). All of the other languages will also get a rewrite in the following days, so expect more updates.

As for a Brackets version, currently I don't have plans for it since I mostly use Atom, but hopefully someone else can make it because it is a nice theme.

**Comment by dindrala** [comment]
@gt-rios wow it looks great! thank you so much for the fast turn around time on it. 

Yes, I agree, it's really is a nice theme.

@Denisov21 
@larz0
@MarcelGerber
@MatthewTRader
@MiguelCastillo
@rpereira
@wormeyman

Would any of you on the brackets team be open to building this theme for brackets? It is the Quantaum Firefox browser theme.

**Comment by wormeyman** [comment]
Wow way cool I have moved on to vscode but hopefully someone here can help

**Comment by dindrala** [comment]
@wormeyman Yeah this theme is amazing. Fingers crossed!

**Comment by dindrala** [comment]
@Denisov21
@larz0
@MarcelGerber
@MatthewTRader
@MiguelCastillo
@rpereira


Anyone?
